src/components/AnimatedStep/AnimatedStepProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {useState} from 'react';
+import AnimatedStepContext, {AnimationDirection} from './AnimatedStepContext';
+import CONST from '../../CONST';
+import ChildrenProps from '../../types/utils/ChildrenProps';
+
+type AnimatedStepProviderProps = ChildrenProps;
```

Comment:
@JKobrynski Do we really need this type?

Reply:
Not really, good catch!